webapp/src/pages/Map.tsx

```diff
@@ -386,6 +474,17 @@ function MapPage() {
     }))
   }, [geoMsgData, msgViewerSliderStartDate, msgViewerSliderEndDate, activeLayers, filter])
 
+  // Effect for handling point source updates moove-ai-layer
+  useEffect(() => {
+    // if the moove-ai-layer is not active, exit the effect
+    if (!activeLayers.includes('moove-ai-layer')) return
+
+    setMooveAiDataSource((prevDataSource) => ({
+      ...prevDataSource,
+      features: mooveAiData,
+    }))
+  }, [mooveAiData, activeLayers, mooveAiFilter])
```

Comment:
does the mooveAiDataSource need to exist? Can we not just reference mooveAiData directly from the rendered element?

Reply:
Addressed